State the the write permission is for VM

andynava opened the pull request:
I believe that the user needs to have write permission on the vm, so instead of saying that the user needs to have "write access for the Azure resource", I think it makes more sense to explicitly state that it requires "write access for the VM"

When I first read it, I wasn't sure what "Azure resource" referred to.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
State the the write permission is for VM

I believe that the user needs to have write permission on the vm, so instead of saying that the user needs to have "write access for the Azure resource", I think it makes more sense to explicitly state that it requires "write access for the VM"
